package.json

```diff
@@ -156,7 +156,7 @@
     "test:type": "yarn workspaces foreach -piv --include '@uppy/*' --exclude '@uppy/{angular,react-native,locales,companion,provider-views,robodog,svelte}' exec tsd",
     "test:unit": "yarn run build:lib && yarn test:watch",
     "test:watch": "vitest --environment jsdom --dir packages/@uppy",
-    "test": "npm-run-all lint test:locale-packs:unused test:unit test:type test:companion",
+    "test": "CI=true npm-run-all lint test:locale-packs:unused test:unit test:type test:companion",
```

Comment:
```suggestion
    "test": "npm-run-all lint test:locale-packs:unused test:unit test:type test:companion",
```

Reply:
This was deliberate, because the test command hangs forever locally if we don’t do this (vitest runs in watch mode, see the commit)